doc/user-manual/getting-started/installation.rst

```diff
@@ -174,23 +174,34 @@ Nix or NixOS
 ------------
 
 Agda is part of the Nixpkgs collection that is used by
-https://nixos.org/nixos. To install Agda and agda-mode for Emacs,
-type:
+https://nixos.org/nixos. There are two ways to install Agda from nix:
 
-.. code-block:: bash
+* As Agda is a Haskell package available from hackage you can install
+  it like any other haskell package. To install Agda and agda-mode for
+  Emacs, type:
+
+  .. code-block:: bash
+
+    nix-env -f "<nixpkgs>" -iA haskellPackages.Agda
+
+* If you are tracking ``nixos-unstable`` or ``nixpkgs-unstable`` (the
+  default on MacOS) or are using a version of NixOS newer than 20.03
+  then you should be able to install via:
+
+  .. code-block:: bash
 
-  nix-env -f "<nixpkgs>" -iA haskellPackages.Agda
+    nix-env -i agdaWithPackages
 
-If you’re just interested in the library, you can also install the
-library without the executable. The Agda standard library is currently
-not installed automatically.
+  This version makes using and developing Agda libraries with nix
```

Comment:
Referring to the nix manual doesn't seem like a bad idea here to me

Reply:
I agree with @alexarice. I think this is very useful info but I would rather keep it short and link to the the manual - it will remain up-to-date and also having a section dedicated to agda makes it very suitable.

Reply:
I tried this:`nix-env -f "<nixpkgs>" -iE "nixpkgs: (nixpkgs {}).agda.withPackages (agdaPackages: [ agdaPackages.standard-library ])"` which worked for me, thanks @turion . But, where does it put the standard library/how to I refer to it in my `~/.agda/libraries` file?

Reply:
The point is that you don't need a `libraries` file anymore. `nix` manages it for you. Also, you don't need to know where the standard library is. It's somewhere in `/nix/store`, and the correct path is known by Agda.

Reply:
> The point is that you don't need a `libraries` file anymore. `nix` manages it for you. Also, you don't need to know where the standard library is. It's somewhere in `/nix/store`, and the correct path is known by Agda.

Oh! I am learning a lot doing this :) But, I cannot get it to work :(

1. I moved my existing `~/.agda` folder out of the way.
2. I installed agda using `nix-env -f "<nixpkgs>" -iE "nixpkgs: (nixpkgs {}).agda.withPackages (agdaPackages: [ agdaPackages.standard-library ])"`
3. I opened emacs and created this test file that uses a standard library module:
```
open import Data.Nat
```
4. Agda cannot find Data.Nat:
```
/private/tmp/test.agda:1,1-21
Failed to find source of module Data.Nat in any of the following
locations:
  /private/tmp/Data/Nat.agda
  /private/tmp/Data/Nat.lagda
  /nix/store/fsqq93r1l8gwl8d0c9ly5isd85m0aqxc-Agda-2.6.1-data/share/ghc-8.8.4/x\
86_64-osx-ghc-8.8.4/Agda-2.6.1/lib/prim/Data/Nat.agda
  /nix/store/fsqq93r1l8gwl8d0c9ly5isd85m0aqxc-Agda-2.6.1-data/share/ghc-8.8.4/x\
86_64-osx-ghc-8.8.4/Agda-2.6.1/lib/prim/Data/Nat.lagda
when scope checking the declaration
  open import Data.Nat
```

Reply:
You still need the `defaults` file or an `.agda-lib` file at project root

Reply:
ah, thanks, that works.